cypress.json

```diff
@@ -1,5 +1,8 @@
 {
   "projectId": "qa7fe2",
   "ignoreTestFiles": "*.js",
-  "baseUrl": "http://localhost:3000"
+  "baseUrl": "http://localhost:3000",
+  "env": {
+    "RETRIES": 2
+  }
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-21T14:14:45Z" title="Friday, February 21st 2020, 3:14:45 pm +01:00">Feb 21, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,5 +1,8 @@
 {
   "projectId": "qa7fe2",
   "ignoreTestFiles": "*.js",
-  "baseUrl": "http://localhost:3000"
+  "baseUrl": "http://localhost:3000",
+  "env": {
+    "RETRIES": 2
+  }
```


Also, not related to `firefox` as a test runner, but unless I'm missing something, does anyone think `retries` are actually occurring? hopefully, we can see some evidence that this works